src/shared/components/case-viewer/services/event-trigger.resolver.ts

```diff
@@ -40,6 +42,7 @@ export class EventTriggerResolver implements Resolve<CaseEventTrigger> {
     if (-1 === EventTriggerResolver.IGNORE_WARNING_VALUES.indexOf(ignoreWarning)) {
       ignoreWarning = 'false';
     }
+    this.profileService.get().subscribe(_ => this.profileNotifier.announceProfile(_));
```

Comment:
If this profile call truly needs to be done before the case loading, I think it has to be pipe'd in. Maybe using a forkJoin with the case service call.

Reply:
I have replaced the http calls with forkJoin